src/game/common/globaldata.h

```diff
@@ -226,13 +226,12 @@ class GlobalData : public SubsystemInterface
     bool m_displayDebug; // not 100% sure and needs confirming
     bool m_winCursors;
     bool m_constantDebugUpdate;
+    bool m_benchMark;
```

Comment:
m_showTeamDot and the stats ones need checking, there are in fact two bools, see wb 00687FB2

Reply:
fixed.